### PR TITLE
Fix for Upload to Github releases stopped working #894

### PR DESCRIPTION
**Pull request by naveensrinivasan**
Fix for Upload to Github releases stopped working #894. The `ExpandUriTemplate` could handle only one parameter which is the bug.

### CODE REVIEW

File: Octokit.Tests/Helpers/StringExtensionsTests.cs [hunk 1]
```diff
@@ -57,9 +57,11 @@ public class TheExpandUriTemplateMethod
             [InlineData("https://host.com/path?name=other", "https://host.com/path?name=other")]
             [InlineData("https://host.com/path?name=example name.txt", "https://host.com/path{?name}")]
             [InlineData("https://host.com/path", "https://host.com/path{?other}")]
+            [InlineData("https://host.com/path?name=example name.txt,label=labeltext", "https://host.com/path{?name,label}")]
+            [InlineData("https://host.com/path?name=example name.txt,label=labeltext", "https://host.com/path{?name,label,other}")]
```
Comment (Haacked) [inline Octokit.Tests/Helpers/StringExtensionsTests.cs]: Is that the proper expansion? Or should it be `?name=example name.txt&label=labeltext`

**Comment by Haacked** [comment]
:cool: Did you test this out?

**Comment by naveensrinivasan** [comment]
Yes, I checked in the test before the actual fix that was causing the CI to fail the build.

**Comment by Haacked** [comment]
> Yes, I checked in the test before the actual fix that was causing the CI to fail the build.

Ok, I just wanted to make sure you actually tested the Upload to Releases with this fix.

**Comment by naveensrinivasan** [comment]
Yes, I tested the Upload to releases with this fix.

https://github.com/naveensrinivasan/TestUpload/releases

**Comment by Haacked** [comment]
selfie-0

**Comment by Haacked** [comment]
Thanks!!!

**Comment by forki** [comment]
Very cool. Are you going to release this as a hot fix? It's blocking the whole F# ecosystem at the moment ;-)

**Comment by Haacked** [comment]
> It's blocking the whole F# ecosystem at the moment

Yikes! I'll try and get a release out soon.

**Comment by forki** [comment]
Yes. Thanks to projectscaffold project everybody uses Octokit. ;-)

Thanks again to both of you. Really appreciate this.

**Comment by Haacked** [comment]
@forki cool, got a sec to sanity check the release notes here? https://github.com/octokit/octokit.net/pull/898

**Comment by forki** [comment]
Lol. Made my day.

**Comment by Haacked** [comment]
This is now [live on NuGet](https://www.nuget.org/packages/octokit) as of version `0.16.0`!

**Comment by forki** [comment]
works like a charme. thanks so much. 

free

**Pull request merged**